uaclient/tests/test_ua_timer.py

```diff
@@ -183,7 +183,7 @@ def test_run_jobs_tries_to_create_new_file_when_invalid(
             assert [
                 mock.call(m_jobs_state())
             ] == fake_file.write.call_args_list
-            assert 2 == m_run_job.call_count
+            assert 3 == m_run_job.call_count
```

Comment:
But don't we have only two jobs ? Why are we running it three times ?

Reply:
we have 3 now :) from https://github.com/canonical/ubuntu-advantage-client/pull/2409